examples/vercel-ai-chatbot/lib/hooks/use-vellum-chat.tsx

```diff
@@ -22,7 +21,6 @@ const useVellumChat = ({
   onFunctionCall?: (functionCall: FulfilledFunctionCall) => Promise<unknown>
 }) => {
   const id = React.useMemo(() => chatId ?? nanoid(), [chatId])
-  const path = usePathname()
```

Comment:
this was a stale value, causing the clearing of the chat. We instead fetch from the dom directly